lib/src/components/card_rounded.dart

```diff
@@ -49,20 +66,33 @@ class CardRounded extends StatelessWidget {
       child: Container(
         height: height,
         width: width,
-        padding: const EdgeInsets.symmetric(
-          vertical: 12,
-          horizontal: 12,
-        ),
+        padding: padding ??
+            const EdgeInsets.symmetric(
+              vertical: 12,
+              horizontal: 12,
+            ),
+        child: child,
       ),
     );
 
     if (onClose != null) {
-      const offset = -5.0;
-      _closeButton = Positioned(
-        right: offset,
-        top: offset,
-        child: _CloseButton(onTap: onClose),
-      );
+      if (closeButtonAlignment == Alignment.topRight) {
+        _closeButton = Positioned(
+          right: _defaultOffset,
+          top: _defaultOffset,
+          child: _CloseButton(onTap: onClose, iconData: customButtonIcon),
+        );
+      } else if (closeButtonAlignment == Alignment.bottomRight) {
+        _closeButton = Positioned(
+          right: _defaultOffset,
+          bottom: _defaultOffset,
+          child: _CloseButton(onTap: onClose, iconData: customButtonIcon),
+        );
```

Comment:
The button is the same here, only the offset changes -- can we have only the logic-dependent code inside the conditional here, and keep the non-changing parts out?